replication.proto

```diff
@@ -0,0 +1,155 @@
+
+syntax = "proto3";
+package replication;
+
+import "google/protobuf/descriptor.proto";
+
+option go_package = ".;replication";
```

Comment:
This option value needs to be changed to generate a lib file in `lib/go` as of  now it will generate in the top directory itself

Reply:
`protoc --go_out=lib/go/ *.proto` This will be added to generate the replication.pb.go file in lib/go in the makefile. 
idea is to have a common folder path in the makefile, not in the proto file.